avocado/core/test.py

```diff
@@ -717,6 +717,21 @@ class TimeOutSkipTest(Test):
 
     _skip_reason = "Test skipped due a job timeout!"
 
+    def __init__(self, *args, **kwargs):
```

Comment:
Feels like we should have a SkipTest(Test) with this __init__() and the others inheriting from SkipTest:
 - TimeOutSkipTest(SkipTest)
 - DryRunTest(SkipTest)
 - ReplaySkipTest(SkipTest)